scripts/bootstrap.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -o pipefail
+set -o nounset
+set -o errexit
+
+err_report() {
+  echo "Exited with error on line $1"
+}
+trap 'err_report $LINENO' ERR
```

Comment:
We need single quote to prevent $LINENO from being expanded when the trap line is first parsed

ref: https://stackoverflow.com/questions/6697753/difference-between-single-and-double-quotes-in-bash

https://unix.stackexchange.com/questions/39623/trap-err-and-echoing-the-error-line